yongshin/app.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+dotenv.config();
+
+const { DataSource } = require('typeorm');
+
+const appDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+});
+
+appDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!");
+    })
+    .catch((err) => {
+        console.error("Error during Data Source initialization", err)
+    appDataSource.destroy()
+    })
+
+const app = express();
+const PORT = process.env.PORT;
+
+app.use(cors());
+app.use(morgan('dev'));
+app.use(express.json());
+
+// health check
+app.get("/ping", (req, res) => {
+    res.status(200).json({ "message" : "pong" });
+});
+
+app.post("/signup", async (req, res, next) => {
+    const { name, email, profile_image, password } = req.body
```

Comment:
자바스크립트는 camelCase를 통해 띄어쓰기를 구분하는 컨벤션이 있습니다.

데이터베이스의 컬럼을 작성하는 부분이 아니라면 camelCase로 통일할 수 있도록 수정해주세요!

```suggestion
    const { name, email, profileImage, password } = req.body
```